scripts/gcp_caper_server/create_instance.sh

```diff
@@ -0,0 +1,314 @@
+#!/bin/bash
+set -e
+
+if [[ $# -lt 1 ]]; then
+  echo "Automated shell script to create Caper server instance with PostgreSQL on Google Cloud."
+  echo
+  echo "Usage: ./create_instance.sh [INSTANCE_NAME] [GCP_PRJ]"
+  echo "                            [GCP_SERVICE_ACCOUNT_SECRET_JSON_FILE] [GCP_OUT_DIR]"
+  echo "                            <OPTIONAL_ARGUMENTS>"
+  echo
+  echo "Positional arguments:"
+  echo "  [INSTANCE_NAME]: New instance's name."
+  echo "  [GCP_PRJ]: Your project's ID on Google Cloud Platform. --gcp-prj in Caper."
+  echo "  [GCP_SERVICE_ACCOUNT_KEY_JSON_FILE]: Service account's secret key JSON file. --gcp-service-account-key-json in Caper."
+  echo "  [GCP_OUT_DIR]: gs:// bucket dir path for outputs. --gcp-out-dir in Caper."
+  echo
+  echo "Optional arguments for Caper:"
+  echo "  -l, --gcp-loc-dir: gs:// bucket dir path for localization."
+  echo "  --gcp-region: Region for Google Life Sciences API. us-central1 by default. CHECK SUPPORTED REGIONS. This is different from --zone, which is used for instance creation only. us-central1 by default."
+  echo "  --postgresql-db-ip: localhost by default."
+  echo "  --postgresql-db-port: 5432 by default."
+  echo "  --postgresql-db-user: cromwell by default."
+  echo "  --postgresql-db-password: cromwell by default."
+  echo "  --postgresql-db-name: cromwell by default."
+  echo
+  echo "Optional arguments for instance creation (gcloud compute instances create):"
+  echo "  -z, --zone: Zone. Check available zones: gcloud compute zones list. us-central1-a by default."
+  echo "  -m, --machine-type: Machine type. Check available machine-types: gcloud compute machine-types list. n1-standard-4 by default."
+  echo "  -b, --boot-disk-size: Boot disk size. Use a suffix for unit. e.g. GB and MB. 100GB by default."
+  echo "  --boot-disk-type: Boot disk type. pd-standard (Standard persistent disk) by default."
+  echo "  --image: Image. Check available images: gcloud compute images list. ubuntu-1804-bionic-v20200716 by default."
+  echo "  --image-project: Image project. ubuntu-os-cloud by default."
+  echo "  --startup-script: Startup script CONTENTS (NOT A FILE). These command lines should sudo-install Java, PostgreSQL, Python3 and pip3. DO NOT INSTALL CAPER HERE. some apt-get command lines by default."
+  echo
+
+  if [[ $# -lt 4 ]]; then
+    echo "Define all positional arguments."
+  fi
+  exit 1
+fi
+
+# parse opt args first.
+POSITIONAL=()
+while [[ $# -gt 0 ]]; do
+key="$1"
```

Comment:
Would prefer to see the stuff within the `while` loop indented:
```suggestion
    key="$1"
```

Reply:
Done as you suggested.